app/entities/account.go

```diff
@@ -0,0 +1,119 @@
+package entities
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Represents a user account in the digital bank.
+type Account struct {
+	id         string
+	name       string
+	cpf        string
+	secret     string
+	balance    int64
+	created_at time.Time
```

Comment:
Em Go nós temos o costume de usar camelCase ou PascalCase:
```suggestion
	createdAt time.Time
```